pkg/lwip/include/netdb.h

```diff
@@ -0,0 +1 @@
+#include "lwip/netdb.h"
```

Comment:
Maybe this and the other one can be a symlink instead? It seems they are already used in the tree.